src/libs/actions/IOU.js

```diff
@@ -1065,6 +1065,7 @@ function getPayMoneyRequestParams(chatReport, iouReport, recipient, paymentMetho
             chatReportID: chatReport.reportID,
             reportActionID: optimisticIOUReportAction.reportActionID,
             paymentMethodType,
+            policyID: chatReport.chatType === CONST.REPORT.CHAT_TYPE.POLICY_EXPENSE_CHAT ? iouReport.policyID : null,
```

Comment:
This needs to be the `policyExpenseChatID`,  I will add it to the PHP too https://github.com/Expensify/Auth/blob/4289dc6f19d542768fc247553ad06f7ccce8a405/auth/command/PayMoneyRequest.cpp#L105-L106